bundle/src/main/java/com/adobe/acs/commons/httpcache/config/impl/ResourcePropertiesHttpCacheConfigExtension.java

```diff
@@ -35,7 +35,14 @@
 import org.osgi.service.metatype.annotations.Designate;
 import org.osgi.service.metatype.annotations.ObjectClassDefinition;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.apache.commons.collections.CollectionUtils.containsAny;
```

Comment:
static import of methods is unclear and makes it appear these methods are implemented by this class. better to clearly qualify where they come from in code.